source/patterns/@aws-solutions-constructs/aws-lambda-sagemakerendpoint/test/aws-lambda-sagemakerendpoint.test.ts

```diff
@@ -0,0 +1,828 @@
+/**
```

Comment:
Suggest to reduce the no. of snapshots from 7 to 2 or less, to cover the most important scenarios. The rest of test cases can be converted from snapshot to assertion test.

Reply:
Will do.